ci/testing.yaml

```diff
@@ -19,3 +19,4 @@ dependencies:
   - matplotlib
   - pip:
     - methodtools
+    - lunarsky>=0.2.5
```

Comment:
In addition to adding the dependency here, it should be added into the `pyproject.toml` as a dependency